tests/test_ipr/test_upload.py

```diff
@@ -221,6 +227,16 @@ def test_copy_variants_loaded(self, loaded_reports) -> None:
         async_section = get_section(loaded_reports["async"], "copy-variants")
         assert compare_sections(section, async_section)
 
+    # # Uncomment when signatureVariants are supported in pori_ipr_api
```

Comment:
Hi @mathieulemieux 
Would this work on the current devipr or is this a future version?